cmd/connectivity-registry/main.go

```diff
@@ -22,14 +22,21 @@ import (
 
 func main() {
 	var (
-		tlsCertPath string
-		tlsKeyPath  string
-		port        int
+		tlsCertPath                                   string
+		tlsKeyPath                                    string
+		port                                          int
+		orphanImportedServiceRecordDeleteDelaySeconds int
 	)
 
 	flag.StringVar(&tlsCertPath, "tls-cert", "", "the path to the server TLS certificate")
 	flag.StringVar(&tlsKeyPath, "tls-key", "", "the path to the server TLS key")
 	flag.IntVar(&port, "port", 8000, "the serving port for the hamlet server")
+	flag.IntVar(
+		&orphanImportedServiceRecordDeleteDelaySeconds,
+		"orphan-imported-service-record-delete-delay-seconds",
+		5*60,
```

Comment:
Is there a written justification for why this is 5 minutes?

Reply:
We have no justification, it's totally arbitrary. Do you have a preferred delay duration?

Reply:
Nope, I was just wondering how we landed on this value. :)P

Reply:
how about `stale-import-cleanup-delay` as a [DurationVar](https://golang.org/pkg/flag/#DurationVar)